doc/user/CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+This doc is a WIP. There is more useful information in the [README](./README.md).
+
+# Set up
+
+Rather than work on `github.com/materializeinc/materialize` itself, we recommend
+working on your own fork of Materialize; this makes branches much tidier.
+branches tidier.
+
+1. Fork `github.com/materializeinc/materialize`. You can do this by going to
+   that address and clicking **Fork**.
+1. Move to the directory where you want to work, and clone your fork to your
+   local machine:
+
+    ```shell
+    git clone git@github.com:<you>/materialize.git
+    ```
+
+1. Move into the `materialize` directory:
+
+    ```shell
+    cd materialize
+    ```
+
+1. Set `materializeinc/materialize` as your `upstream` remote:
+
+    ```shell
+    git remote add upstream git@github.com:MaterializeInc/materialize.git
+    ```
+
+This will give you your own workspace to use while still giving you the ability
+to `git pull` the latest version of Materialize.
+
```

Comment:
I think it's worth linking to `../developer/guide.md` for detailed work instructions. It's fairly long, so I personally would caveat that folks don't need to read the whole thing if they just want to issue a drive-by PR, maybe something like:

```markdown
We follow a standard github pull-request workflow. For details on our standards and workflow please consult [the developer guide](../developer/guide.md).
```

Reply:
Ah, this is for contributing to the docs; will make that clearer.

Reply:
oh, lol, that makes sense! I *was* confused.